pwndbg/commands/context.py

```diff
@@ -84,6 +84,23 @@ def clear_screen(out=sys.stdout) -> None:
     "context-sections",
     "regs disasm code ghidra stack backtrace expressions threads heap-tracker",
     "which context sections are displayed (controls order)",
+    choices=[
+        "regs",
+        "disasm",
+        "code",
+        "ghidra",
+        "stack",
+        "backtrace",
+        "expressions",
+        "threads",
+        "heap-tracker",
+        "''",
+        '""',
+        "none",
+        "empty",
+        "-",
+        "",
```

Comment:
I think we added `''`, `""`, `-`, `none` and `empty` here bcoz users could not know the available options. We probably don't need all those options if an info is displayed on avaialble options etc.